tests/contenttypes_tests/test_management.py

```diff
@@ -63,3 +70,12 @@ def test_unavailable_content_type_model(self):
         with self.assertNumQueries(0):
             contenttypes_management.create_contenttypes(self.app_config, interactive=False, verbosity=0, apps=apps)
         self.assertEqual(ContentType.objects.count(), self.before_count + 1)
+
+    def test_all_models_removed(self):
+        ContentType.objects.create(app_label='app_with_no_models', model='Fake')
+        with mock.patch('builtins.input', return_value='yes'):
+            with captured_stdout() as stdout:
+                call_command('remove_stale_contenttypes', verbosity=2)
+
+            self.assertIn("Deleting stale content type", stdout.getvalue())
```

Comment:
Use single quotes unless the string contains a single quote.  Check for the correct content type that's deleted?